doc/releases/migration-guide-4.2.rst

```diff
@@ -164,6 +162,24 @@ Display
   the format expected by Zephyr. This change ensures proper behavior of both
   display and video capture samples.
 
+Eeprom
```

Comment:
This should be all caps since it is an acronym for "electrically erasable programmable read-only memory"